doc/tutorial/extending_theano.txt

```diff
@@ -259,7 +259,7 @@ Op Example
 
         def make_node(self, x):
             # check that the theano version has support for __props__
-            assert hasattr(self, '_props')
+            assert hasattr(self, '__props__')
```

Comment:
I think 1 year after 0.7. We'll see the number at that time
Le 25 juin 2015 14:01, "abergeron" <notifications@github.com> a écrit :

> In doc/tutorial/extending_theano.txt
> <https://github.com/Theano/Theano/pull/3061#discussion_r33285114>:
>
> > @@ -259,7 +259,7 @@ Op Example
> >
> >          def make_node(self, x):
> >              # check that the theano version has support for __props__
> > -            assert hasattr(self, '_props')
> > +            assert hasattr(self, '__props__')
>
> You mean 0.7.1 or 0.8 sometime next year?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/3061/files#r33285114>.
>